docs/getting_started/basic/zero-shot-prediction/zero-shot-classification.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>method. We are going to start by downloading a very standard zero-shot-model, CLIP (link to clip):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>